lib/prefab/config_client_presenter.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Prefab
+  class ConfigClientPresenter
+    def initialize(size:, source:, project_id:, project_env_id:, api_key:)
+      @size = size
+      @source = source
+      @project_id = project_id
+      @project_env_id = project_env_id
+      @api_key = api_key
+    end
+
+    def to_s
+      "Configuration Loaded count=#{@size} source=#{@source} project=#{@project_id} project-env=#{@project_env_id} prefab.api-key-id=#{@api_key}"
```

Comment:
Is the api_key here just the ID? or the whole thing. We shouldn't print the api key.

Reply:
ok, updated to extract the id from the API key.